ophidian/placement/Library.h

```diff
@@ -0,0 +1,66 @@
+#ifndef LIBRARY_H
+#define LIBRARY_H
+
+#include <ophidian/entity_system/EntitySystem.h>
+#include <ophidian/entity_system/Property.h>
+#include <ophidian/standard_cell/StandardCells.h>
+#include <ophidian/geometry/Models.h>
+#include <ophidian/util/Units.h>
+
+namespace ophidian {
+namespace placement {
+class Library
```

Comment:
I agree with you. But to be consistent, let us use Library for all DB concepts. The namespace will differentiate. Thus rename Timing_Library to Library.